code/Test_definitions/connected-network-type-subscriptions.feature

```diff
@@ -1,289 +1,498 @@
-@Connected_network_type_subscription
-Feature: CAMARA Connected Network Type Subscriptions API, vwip - Operations on subscriptions
+Feature: CAMARA Connected Network Type Subscriptions API, vwip - Operations to manage connected network type subscriptions
 
-# Input to be provided by the implementation to the tests
-# References to OAS spec schemas refer to schemas specifies in connected-network-type-subscriptions.yaml, version vwip
+  # Input to be provided by the implementation to the tester
+  #
+  # Implementation indications:
+  # * List of device identifier types which are not supported, among: phoneNumber, networkAccessIdentifier, ipv4Address, ipv6Address
+  #
+  # Testing assets:
+  # * A device object where the tester can select between multiple network types.
+  # * (optional: Additional devices object which supports 2G, 3G, 4G and/or 5G)
+  # * The known connected Generation of Mobile Communication Technology.
+  # * A sink-url identified as "callbackUrl", which receives notifications
+  #
+  # References to OAS spec schemas refer to schemas specifies in connected-network-type-subscriptions.yaml
 
   Background: Connected Network Type Subscriptions setup
-    Given the resource "{apiroot}/connected-network-type-subscriptions/vwip" as base-url
+    Given the resource "{apiroot}/connected-network-type-subscriptions/vwip/subscriptions" as base-url
     And the header "Authorization" is set to a valid access token
     And the header "x-correlator" is set to a UUID value
 
-######### Happy Path Scenarios #################################
+##########################
+# Happy path scenarios
+##########################
 
-  @connected_network_type_subscriptions_01_create_connected_network_type_subscription_sync
-  Scenario: Create connected network type subscription synchronously
-    Given that subscriptions are created synchronously
-    And a valid subscription request body
-    When the request "createConnectedNetworkTypeSubscription" is sent
-    Then the response code is 201
+  @connected_network_type_subscriptions_01_sync_creation
+  Scenario Outline: Create connected network type subscription synchronously
+    Given use BaseURL
+    When the HTTP "POST" request is sent
```

Comment:
A general guideline for all scenarios from the "API-Testing-Guidelines.md" is the use of the "operationId"
```yaml
Request sending

For scenarios testing individual operations, the request will be sent by means of a single When step, indicating the operationId specified in the spec, as the operationId uniquely identifies the HTTP method and resource path. E.g.:

When the request "{operationId}" is sent

```